.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+## PR Checklist
+<!-- Please check if your PR fulfills the following requirements: -->
+
+- [ ] Added label to the Pull Request for easier discoverability and search
+- [ ] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
+- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)
```

Comment:
as this doc would be public, would be nice to have a checklist reminding the author that they are not sharing any Employer confidential details (specifically to gate Intel confidential details) eg
[ ] No Employer Confidential details are added

Reply:
Added